filament/backend/src/vulkan/platform/VulkanPlatform.cpp

```diff
@@ -716,6 +733,7 @@ Driver* VulkanPlatform::createDriver(void* sharedContext,
     // Store the extension support in the context
     context.mDebugUtilsSupported = setContains(instExts, VK_EXT_DEBUG_UTILS_EXTENSION_NAME);
     context.mDebugMarkersSupported = setContains(deviceExts, VK_EXT_DEBUG_MARKER_EXTENSION_NAME);
+    context.mMultiviewSupported = setContains(deviceExts, VK_KHR_MULTIVIEW_EXTENSION_NAME);
```

Comment:
on line 728, we should remove the multiview extension if `driver.config.stereoType != MULTIVIEW`